apps/web/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "start": "pnpm panda --watch & cross-env NODE_OPTIONS=--max_old_space_size=8192 PORT=4200 react-app-rewired start",
     "prebuild": "rimraf build",
-    "build": "pnpm panda && cross-env NODE_OPTIONS=--max_old_space_size=4096 GENERATE_SOURCEMAP=false react-app-rewired --max_old_space_size=4096 build",
+    "build": "pnpm panda && cross-env NODE_OPTIONS=--max_old_space_size=8192 GENERATE_SOURCEMAP=false react-app-rewired --max_old_space_size=8192 build",
```

Comment:
Without this the build would fail with the following error:
```
FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
```
Since this setting is at the value of `8192` in the start command, I adjusted it here and then the build passed.